src/App.tsx

```diff
@@ -1,223 +1,189 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
 import { AuthContext } from './components/Auth/AuthContext';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import {
+  ErrorNotification,
+} from './components/ErrorNotification/ErrorNotification';
+
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodoOnServer,
+} from './api/todos';
+
+import {
+  getCompletedTodoIds,
+  getFilterTodos,
+} from './components/helperFunction';
+import { useError } from './controllers/useError';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempNewTodo, setTempNewTodo] = useState<Todo | null>(null);
+  const [completedFilter, setCompletedFilter] = useState<Filter>(Filter.all);
```

Comment:
Regularly, we start the enum key with the capital letter
```suggestion
  const [completedFilter, setCompletedFilter] = useState<Filter>(Filter.All);
```